### PR TITLE
OCM-9288 | test: automate id:60688 reuse operator-roles and oidc config

### PR DESCRIPTION
**Pull request by yuwang-RH**
[OCM-9288](https://issues.redhat.com//browse/OCM-9288) | test: automate id:60688 reuse operator-roles and oidc config

Log:https://privatebin.corp.redhat.com/?7e86348b7c31c68e#GEvGyKHSSY3tk9F97CZbya3bfCnjMnrcPzhCUeyCHWv1

### CODE REVIEW

File: tests/e2e/test_rosacli_cluster.go [hunk 1]
```diff
@@ -1743,3 +1744,143 @@ var _ = Describe("create/delete operator-roles and oidc-provider to cluster",
...
+			"-y",
+		)
+		Expect(err).To(BeNil())
+		textData := rosaClient.Parser.TextData.Input(output).Parse().Tip()
+		Expect(textData).To(ContainSubstring("Successfully deleted the OIDC provider"))
+
+	})
+
+	It("to reuse operator-roles prefix and oidc config - [id:60688]",
+		labels.Critical, labels.Runtime.Day2,
```
Comment (yasun1) [inline tests/e2e/test_rosacli_cluster.go:1804]: Is it a critical scenario in the customer usage?

Comment (yuwang-RH) [reply tests/e2e/test_rosacli_cluster.go:1804]: Yes it it

File: tests/e2e/test_rosacli_cluster.go [hunk 2]
```diff
@@ -1743,3 +1744,143 @@ var _ = Describe("create/delete operator-roles and oidc-provider to cluster",
...
+		err                      error
+		oidcConfigToClean        string
+		ocmResourceService       rosacli.OCMResourceService
+		originalMajorMinorVerson string
+		clusterConfig            *config.ClusterConfig
+		clusterService           rosacli.ClusterService
+		awsClient                *aws_client.AWSClient
+		operatorPolicyArn        string
+		clusterID                string
+	)
```
[the remark this answers is not in the document]
Comment (yuwang-RH) [reply tests/e2e/test_rosacli_cluster.go:1761]: All common variables are declared in high level and can be used by others cases in the same blocker.
profile is used for the condition of oidc config which is a matrix defined in profile. And clusterConfig is a struct parsing the clusterConfig file in shared_dir, both of them are needed for the case.

**Comment by yuwang-RH** [comment]
@yasun1 Fix the line of expecting err as your comment. Could you help review again?

**Pull request merged**